app/controllers/suggestions_job_statuses_controller.rb

```diff
@@ -0,0 +1,12 @@
+class SuggestionsJobStatusesController < ApplicationController
+  def show
+    @job_status = SuggestionsJobStatus.find(params[:id])
+    if !@job_status.done?
+      render json: {}, status: 200
+    elsif @job_status.save
```

Comment:
It's usually a bad idea to write to the database in a show or index action: https://robots.thoughtbot.com/http-safety-doesnt-happen-by-accident